native/src/routes/Pois.tsx

```diff
@@ -96,7 +96,9 @@ const Pois = ({
         )
         setSelectedFeature(currentFeature ?? null)
       }
-      location && setFeatureLocations(featureLocations)
+      if (location) {
+        setFeatureLocations(featureLocations)
+      }
```

Comment:
@f1sh1918 @Taggotty is this if correct/necessary?

Reply:
Not sure. @f1sh1918 ?